README.md

```diff
@@ -36,22 +36,24 @@ Gnofract 4D requires these packages to run:
 - Python version 3.6 or higher
 - GTK version 4.6 or higher
 - PyGObject version 3.29.2 or higher to access GTK from Python
-- A C++ compiler (used at runtime to compile your fractal formulas)
+- A C compiler (used at runtime to compile your fractal formulas)
 
-On Ubuntu, these can be installed with:
+On Debian/Ubuntu, these can be installed with:
 
     sudo apt install gcc gir1.2-gtk-4.0 python3-gi
 
 To build from source you also need:
 
+- A C++ compiler
 - headers for libpng and libjpeg
 - Python headers
 - glib-compile-resources and optionally xmllint
 - pkg-config
+- Python setuptools package
 
-On Ubuntu, these can be installed with:
+On Debian/Ubuntu, these can be installed with:
 
-    sudo apt install libglib2.0-dev-bin libjpeg-dev libpng-dev libpython3-dev libxml2-utils pkg-config
+    sudo apt install g++ libglib2.0-dev-bin libjpeg-dev libpng-dev libpython3-dev libxml2-utils pkg-config python3-setuptools
```

Comment:
There's a subtle reason why we're suggesting this meta package that isn't obvious from the packages it installs - and that's to do with installation status (automatic, vs manual) and package nomenclature changes (if one day, libc6-dev changes name to, say, libc7-dev, manual installation will break, but they guarantee that this meta package will continue to just work)

It might bring in more than we need, but it's safe, non-brittle, and doesn't wind up with the user being "stuck" with manual installation marked packages if ever they want to remove Gnofract4d and these packages after - meaning apt/dpkg's cleanup commands will work properly.

Reply:
I concur with @dragonmux 's reasoning: we are not wanting a particular c++ compiler; we are wanting standardized build environment for gnofract4d to make use of.  Lets not micro-manage our users systems, but let it upto the distribution managers to select what exactly should be in a basic build environment.